packages/db/src/core/utils.ts

```diff
@@ -28,21 +27,4 @@ export function getDbDirectoryUrl(root: URL | string) {
 	return new URL('db/', root);
 }
 
-export function defineDbIntegration(integration: AstroDbIntegration): AstroIntegration {
-	return integration;
-}
```

Comment:
Moved this straight to the user-facing utils library to avoid importing it from here where Vite is also imported